torch/utils/_foreach_utils.py

```diff
@@ -5,6 +5,19 @@
 from torch import Tensor
 from torch.autograd.grad_mode import no_grad
 
+__all__ = ["get_foreach_kernels_supported_devices", "get_fused_kernels_supported_devices"]
+
+def get_foreach_kernels_supported_devices() -> List[str]:
+    r"""
+    Return the device type list that supports foreach kernels.
+    """
+    return ["cuda", torch._C._get_privateuse1_backend_name()]
+
+def get_fused_kernels_supported_devices() -> List[str]:
+    r"""
+    Return the device type list that supports fused kernels in optimizer.
+    """
+    return ["cuda", torch._C._get_privateuse1_backend_name()]
```

Comment:
Has torch._C._get_privateuse1_backend_name() already landed as how private backends should be referred to?

Reply:
> Has torch._C._get_privateuse1_backend_name() already landed as how private backends should be referred to?

yes, the API of torch._C._get_privateuse1_backend_name()  has been merged and used many times in pytorch.